File: OneCodeTeam/filter-data-using-jquery-load-16629/[VB]-filter data using jquery load (JqueryFilterLoadDynamicContent)/VB/VBASPNETJqueryFilterLoadDynamicContent/Province.vb
'****************************** Module Header ******************************\
' Module Name:    Province.vb
' Project:        VBASPNETJqueryFilterLoadDynamicContent
' Copyright (c) Microsoft Corporation
'
' Province Data.
'
' This source is subject to the Microsoft Public License.
' See http://www.microsoft.com/en-us/openness/resources/licenses.aspx#MPL.
' All other rights reserved.
'
' THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
' EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
' WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
'*****************************************************************************/

Public Class Province
    Public Property ProvinceName() As String
        Get
            Return m_ProvinceName
        End Get
        Set(ByVal value As String)
            m_ProvinceName = value
        End Set
    End Property
    Private m_ProvinceName As String
    Public Property PinYin() As String
        Get
            Return m_PinYin
        End Get
        Set(ByVal value As String)
            m_PinYin = value
        End Set
    End Property
    Private m_PinYin As String
    Public Property countryID() As String
        Get
            Return m_countryID
        End Get
        Set(ByVal value As String)
            m_countryID = value
        End Set
    End Property
    Private m_countryID As String
End Class

File: OneCodeTeam/filter-data-using-jquery-load-16632/[C#]-filter data using jquery load/C#/CSASPNETJqueryFilterLoadDynamicContent/Province.cs
/****************************** Module Header ******************************\
* Module Name:    Province.cs
* Project:        CSASPNETJqueryFilterLoadDynamicContent
* Copyright (c) Microsoft Corporation
*
* Province data.
* 
* This source is subject to the Microsoft Public License.
* See http://www.microsoft.com/en-us/openness/resources/licenses.aspx#MPL.
* All other rights reserved.
* 
* THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
* EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
* WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\*****************************************************************************/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CSASPNETJqueryFilterLoadDynamicContent
{   
    public class Province
    {
        public string ProvinceName { get; set; }
        public string PinYin { get; set; }
        public string countryID { get; set; }
    }
}
